config/testreport/css/default.css

```diff
@@ -20,6 +20,14 @@ body
     background-color: #f1f1f1;
 }
 
+button#clear-input
```

Comment:
As there are probably more than one such clear-input buttons in the HTML document, this selector should be changed to `button.clear-input` instead (use CSS class instead of ID).